desktop/app_src/src/main.js

```diff
@@ -38,15 +44,14 @@ function createWindow() {
   } else {
     config = { Sizing: { Width: "800", Height: "600", Maximized: false } };
     fs.writeFile(
-      "settings.ini",
+      settingsFile,
```

Comment:
```suggestion
 if (fs.existsSync(settingsFile)) {
    getConfig();
  } else {
    config = { Sizing: { Width: "800", Height: "600", Maximized: false } };
    fs.writeFile(
      settingsFile,
```

I think you forgot to change this path, it will fail to load on Linux as it is.